src/plugins/telemetry/schema/oss_plugins.json

```diff
@@ -202,29 +202,29 @@
         }
       }
     },
-    "search": {
+    "search-session": {
```

Comment:
Why is this a part of this PR?

Reply:
it's just some auto thing, i didn't rename, i think order got changed.